vrecord

```diff
@@ -798,18 +798,20 @@ shift $(( ${OPTIND} - 1 ))
 
 if [[ "${runtype}" = "GUI" ]] ; then
     _master_gui
-elif [[ "${runtype}" = "reset" ]] ; then
+fi
+if [[ "${runtype}" = "reset" ]] ; then
     _report -q -n "Reseting the configuration will clear ${config_file}. Please enter [Y] to confirm: "
     read reset_response
-    if [[ "${reset_response}" = [Yy] ]] ; then
+    if [[ "${reset_response}" = "[Y]" ]] ; then
```

Comment:
I think this should still be "[Yy]"

Reply:
Okay, I wasn't sure since on an older version that I was comparing it to it was the way that i changed it to.

Reply:
Yes, see https://github.com/amiaopensource/vrecord/pull/202#pullrequestreview-83068360.